tests/consensus_tests/fixtures.py

```
@@ -91,6 +91,7 @@ def upsert_random_points(
     ordering="weak",
     with_sparse_vector=True,
     shard_key=None,
+    num_cities=None,
     headers={},
```

Ruff (0.14.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)